applications/SwimmingDEMApplication/python_scripts/navier_stokes_solver_vms_monolithic_DEMCoupled.py

```diff
@@ -57,6 +59,21 @@ def _SetUpVMSDEM(self,settings):
         use_oss = settings["use_orthogonal_subscales"].GetBool()
         self.process_data[KratosMultiphysics.OSS_SWITCH] = int(use_oss)
 
+    def _SetUpDVMSDEM(self,settings):
+        default_settings = KratosMultiphysics.Parameters(r"""{
+            "element_type": "dvmsDEM",
+            "use_orthogonal_subscales": false,
+            "dynamic_tau": 0.0,
+            "element_name": "DVMSDEMCoupled"
+        }""")
+        settings.ValidateAndAssignDefaults(default_settings)
+
```

Comment:
Consider the changes in #7997

Reply:
Why did you include the dynamic_tau in the DVMS settings? It is not used in the element...

Reply:
It is used in the base quasi-static class to calculate the stabilization constants.